server/src/main/java/org/elasticsearch/action/admin/cluster/storedscripts/GetScriptContextResponse.java

```diff
@@ -122,11 +108,11 @@ public boolean equals(Object o) {
             return false;
         }
         GetScriptContextResponse that = (GetScriptContextResponse) o;
-        return contextNames.equals(that.contextNames);
+        return new HashSet<>(contexts).equals(new HashSet<>(that.contexts));
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(contextNames);
+        return Objects.hash(new HashSet<>(contexts));
```

Comment:
Ditto here, we shouldn't be allocating anything in hashCode()

Reply:
Removed allocation.